src/index.html

```diff
@@ -1,17 +1,175 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+      autocomplete="off">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form-field ">
+          Surname:
+          <input
+            class="inp"
+            type="text"
+            name="surname"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="ternOfTheAgreement"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+            class="inp"
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="8"
+            maxlength="32"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field inp">
+          <label>
+            Do you love cats?
+            <input
+              type="radio"
+              name="loveCats"
+              value="Yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="loveCats"
+              value="No"
+            >
+            No
+          </label>
+        </div>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            value="#000"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="timeToBad"
+          >
+        </label>
+
+        <label class="form-field brand">
```

Comment:
redundant class `brand`
```suggestion
        <label class="form-field">
```